app/controllers/issues_controller.rb

```diff
@@ -12,8 +12,12 @@ def new
 
   def show
     @issue = Issue.find(params[:id])
+    category_id = @issue.category_id
```

Comment:
@relevant_issues = @issue.category.issues.where.not(id: issue_id).order('random()').limit(4)